### PR TITLE
fix testdata file

### PR DESCRIPTION
**Pull request by lpantano**
Last bit of the migration, point to the right test data for the current version and update mirtop version container.

## PR checklist

- [X] This comment contains a description of changes (with reason).
- [X] Make sure your code lints (`nf-core lint .`).
- [X] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [X] `CHANGELOG.md` is updated.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # nf-core/smrnaseq: Changelog
 
-## v1.3.0dev - [2021-09-15]
+## v2.0.0dev - [2021-09-15]
```
Comment (JoseEspinosa) [inline CHANGELOG.md:3]: :heart_eyes:

File: conf/test.config [hunk 2]
```diff
@@ -20,21 +20,12 @@ params {
...
- //   mature = 'https://github.com/nf-core/test-datasets/raw/smrnaseq/reference/mature.fa'
- //   hairpin = 'https://github.com/nf-core/test-datasets/raw/smrnaseq/reference/hairpin.fa'
- //   mirna_gtf = 'https://github.com/nf-core/test-datasets/raw/smrnaseq/reference/hsa.gff3'
 
-    input = 'https://github.com/nf-core/test-datasets/raw/smrnaseq-better-input/testdata/samplesheet.csv'
-    fasta = 'https://github.com/nf-core/test-datasets/raw/smrnaseq-better-input/reference/genome.fa'
-    mature = 'https://github.com/nf-core/test-datasets/raw/smrnaseq-better-input/reference/mature.fa'
-    hairpin = 'https://github.com/nf-core/test-datasets/raw/smrnaseq-better-input/reference/hairpin.fa'
-    mirna_gtf = 'https://github.com/nf-core/test-datasets/raw/smrnaseq-better-input/reference/hsa.gff3'
+    input = 'https://github.com/nf-core/test-datasets/raw/smrnaseq/testdata/samplesheet.csv'
```
Comment (JoseEspinosa) [inline conf/test.config:24]: Maybe we could implement a similar versioning such as in the rnaseq pipeline e.g. https://github.com/nf-core/test-datasets/tree/rnaseq/samplesheet/v3.4

**Review by JoseEspinosa — APPROVED** [review]
Just made a small comment regarding the versioning of the sample sheet, otherwise looks good to me :candy:

File: CHANGELOG.md (hunk 1 above)
Comment (apeltzer) [reply CHANGELOG.md:3]: Yes I think a 2.0.0 is justified here!

File: conf/test.config (hunk 2 above)
Comment (apeltzer) [reply conf/test.config:24]: Great idea, especially if some potential future changes are coming in that might have to adjust the format at some point again 👍🏻

**Review by apeltzer — APPROVED** [review]

File: conf/test.config (hunk 2 above)
Comment (lpantano) [reply conf/test.config:24]: ok, should I merge and then coordinate this? or do you want to do it now?

**Comment by JoseEspinosa** [comment]
I merge this PR. A new PR will address the requested changes regarding test-datasets versioning 🚀

**Pull request merged**